imagetest/fixtures.go

```diff
@@ -259,6 +272,16 @@ func (t *TestVM) AddAliasIPRanges(aliasIPRange, rangeName string) error {
 	return nil
 }
 
+// AddPrivateIP set IPv4 internal IP address to assign to the current test VMs.
+func (t *TestVM) AddPrivateIP(networkIP string) error {
+	if t.instance.NetworkInterfaces == nil {
+		return fmt.Errorf("Must call SetCustomNetwork prior to AddPrivateIP")
```

Comment:
I'd suggest against this, unless you have a strong usecase: 
 1. It implies a before/after relationship of calling methods
 2. You update the first network interface, even though "SetCustomNetworkWithNetworkIP" is currently appending to the list of network interfaces.

Reply:
This method actually is not called in current revision. I make a refactor for this to inidcate which network interface as first parameter, then not only for 1st nic.

Reply:
lgtm (I don't have permission to resolve the comment thread)